articles/quickstart/backend/java-spring-security5/01-authorization.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Authorization
+name: How to secure your API using Spring Security 5 and Auth0
+description: This tutorial demonstrates how to add authorization to a an API using Spring Security 5.
+budicon: 500
+topics:
+    - quickstart
+    - backend
+    - java
+    - spring-security
+github:
+    path: 01-Authorization-MVC
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../../../_includes/_api_auth_intro') %>
+
+<%= include('../_includes/_api_create_new') %>
+
+<%= include('../_includes/_api_auth_preamble') %>
+
+## Configure the Sample Project
+
+::: note
+This Quickstart uses Spring MVC. If you are using Spring WebFlux, the steps to secure an API are similar, but some of the implementation details are different. Refer to the [Spring Security WebFlux Sample Code](https://github.com/auth0-samples/auth0-spring-security5-api-sample/tree/master/01-Authorization-WebFlux) to see how integrate Auth0 with your Spring WebFlux API.
+:::
+
+The sample project uses a `/src/main/resources/application.yml` file, which configures it to use the correct Auth0 **Domain** and **API Identifier** for your API. If you download the code from this page it will be automatically configured. If you use the example from GitHub, you will need to fill it in yourself.
+
+```yaml
+auth0:
+  audience: ${apiIdentifier}
```

Comment:
this won't get rendered/replaced when logged in. Are you defining a constants anywhere else?

Reply:
`apiIdentifier` is used by the docs and will do replacement in the code snippet when logged in. The actual example YAML config file in the repo that will do replacement when downloading the example is:

```yaml
debug: true
server:
  port: 3010

auth0:
  # Replace with the API Identifier for your Auth0 API.
  audience: {API_IDENTIFIER}

# The following is standard Spring Security OAuth2 configuration.
spring:
  security:
    oauth2:
      resourceserver:
        jwt:
          # Replace with the domain of your Auth0 tenant.
          # Note the trailing slash is important!
          issuer-uri: https://{DOMAIN}/
```

Reply:
This is new to me. Thanks for sharing!